lib/sentry/event.ex

```diff
@@ -56,6 +58,8 @@ defmodule Sentry.Event do
 
     stacktrace = Keyword.get(opts, :stacktrace, [])
 
+    fingerprint = Keyword.get(opts, :fingerprint, ["{{ default }}"])
```

Comment:
Discussed in-person, but for posterity it seems other Sentry libraries do not send up a key/value for fingerprint if it's nil, so it would probably be best for us to do that too.

Currently we're directly encoding the Event struct to JSON, which is not ideal, and we'll need to not do that to fix the above issue.  Created an issue for it here: #161